Lib/os.py

```diff
@@ -1079,6 +1079,8 @@ class PathLike(abc.ABC):
 
     """Abstract base class for implementing the file system path protocol."""
 
+    __slots__ = ()
```

Comment:
Let's make this change in a separate PR/issue, I think

Reply:
Oh, OK!

Reply:
Sorry if my previous comment was confusing!

I feel like the `.register()` thing works fine in pathlib for now, that this is a separate bugfix that should be considered separately (after we've heard Brett's thoughts), and that the git history will be easier to read if it's its own PR

Reply:
No worries, I was being silly. Issue and PR:

- #104898
- #104899